CLI Reference - Quick typo/edits (#276)
withastro/docs: src/pages/en/reference/cli-reference.md

### `astro dev`

> Runs the Astro development server. This starts an HTTP server that responds to requests for pages stored in `src/pages` (or whichever folder is specified in your [configuration](/en/reference/configuration-reference)).

Reviewer: ```suggestion
Runs the Astro development server. This starts an HTTP server that responds to requests for pages stored in `src/pages` (unless overridden by your `pages` option set in the project [configuration](/en/reference/configuration-reference)).
```

> Runs  Astro's `dev` server. It starts a HTTP server which responds to requests for routes or pages that are specified within `src/pages` directory (unless overridden by your `pages` option set in the project [configuration](/en/reference/configuration-reference)).

Author: ```suggestion
Runs  Astro's `dev` server. It starts an HTTP server which responds to requests for routes or pages that are specified within `src/pages` directory (unless overridden by your `pages` option set in the project [configuration](/en/reference/configuration-reference)).
```

> 

Reviewer: ```suggestion
Builds your site for production based upon your [configuration options](/en/reference/configuration-reference)) and is produced to the `./dist`.
```
